.husky/pre-commit

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-#npm test
+# npm test
```

Comment:
Why comment npm test? Won't we need it later when we add tests?

Reply:
uncomment

Reply:
I don't comment npm test. I just add space. 
![image](https://user-images.githubusercontent.com/76705901/188550969-b52efc0a-33c3-4cab-8b71-10581eb0e75a.png)